docs/config/README.md

```diff
@@ -199,6 +199,10 @@ module.exports = {
 
 ## Build Pipeline
 
+:::tip Configuring CSS Pre-processors
+VuePress comes with built-in webpack config for the CSS pre-processors listed below. For more information on installation these or pre-processors without built-in support, see (Using Pre-Processors)[https://vuepress.vuejs.org/guide/using-vue.html#using-pre-processors] for more information.
+:::
```

Comment:
BTW, you should use relative path of source markdown files (`../guide/using-vue.md#using-pre-processors`) here.